utils/notifications.js

```diff
@@ -574,6 +574,85 @@ const getSiteNotification = async (req, res, authObj) => {
     return res.status(200).json({data, code: 200})
 }
 
+const sendEmailLink = async (req, res) => {
+    if (req.method !== "POST") {
+        return res
+            .status(405)
+            .json(getResponseJSON("Only POST requests are accepted!", 405));
+    }
+    try {
+        const { email, continueUrl } = req.body;
+        const [clientId, clientSecret, tenantId, magicLink] = await Promise.all(
+            [
+                getSecrets(process.env.APP_REGISTRATION_CLIENT_ID),
+                getSecrets(process.env.APP_REGISTRATION_CLIENT_SECRET),
+                getSecrets(process.env.APP_REGISTRATION_TENANT_ID),
+                generateSignInWithEmailLink(email, continueUrl),
+            ]
+        );
+
+        const params = new URLSearchParams();
+        params.append("grant_type", "client_credentials");
+        params.append("scope", "https://graph.microsoft.com/.default");
+        params.append("client_id", clientId);
+        params.append("client_secret", clientSecret);
+
+        const resAuthorize = await fetch(
+            `https://login.microsoftonline.com/${tenantId}/oauth2/v2.0/token`,
+            {
+                method: "POST",
+                headers: {
+                    "Content-Type":
+                        "application/x-www-form-urlencoded;charset=UTF-8",
+                },
+                body: params,
+            }
+        );
+
+        const { access_token } = await resAuthorize.json();
+
+        const body = {
+            message: {
+                subject: `Sign in to Connect for Cancer Prevention Study requested at ${getDatetimeForEmailLink()}`,
```

Comment:
Do we have a decision about time format in subject? UTC time (eg, 20:43 Z for 3:43pm EST) is confusing.

Reply:
I implemented it based on old content from firebase authentication.

Reply:
The emails go to our participants, so they better to be clear. Can you confirm that adding time in this format has been discussed and approved by Ops team?